README.md

```diff
@@ -425,7 +454,15 @@ http_address: 0.0.0.0:8080
 #
 #http_proxy:
 #  url: https://remote-cache.com:8080/cache
-
+#
+#azblob_proxy:
+#  tenant_id: TENANT_ID
+#  storage_account: STORAGE_ACCOUNT
+#  container_name: CONTAINER_NAME
+#  auth_method: client_secret
```

Comment:
I think we should list the allowed auth method values, like we do in the s3 section above